benchmarking/runs/run_concentrate_all_attributes.py

```diff
@@ -0,0 +1,111 @@
+import math
+
+from benchmarking.data.interfaces import MockStudentProviderSettings
+from benchmarking.data.simulated_data.mock_student_provider import (
+    MockStudentProvider,
+)
+from benchmarking.evaluations.metrics.average_gini_index_multi_attribute import (
+    AverageGiniIndexMultiAttribute,
+)
+from benchmarking.evaluations.metrics.num_requirements_satisfied import (
+    NumRequirementsSatisfied,
+)
+from benchmarking.evaluations.metrics.num_teams_meeting_requirements import (
+    NumTeamsMeetingRequirements,
+)
+from benchmarking.evaluations.scenarios.concentrate_all_attributes import (
+    ConcentrateAllAttributes,
+)
+from benchmarking.simulation.simulation import Simulation
+from models.enums import ScenarioAttribute, Gender
+
+
+def run_concentrate_all_attributes():
+    """
+    Goal: Run concentrate all attributes scenario, measure average gini index across all attributes
+    """
+
+    # Defining our changing x-values (in the graph sense)
+    class_sizes = [100, 150, 200, 250, 300]
+    num_trials = 10
+    ratio_of_female_students = 0.5
+    races = [
```

Comment:
For things like this, we can specify it as a ScenarioAttributeValue enum 
(See Gpa, Gender, etc in the enums)

Reply:
Tbh it seems like you only need it for the length anyway so maybe just make them constants?
Like `NUM_RACES`

I think it's worth to do the same for the others too, the min/max doesn't really matter in this case you just need N distinct integer options for each